File: problemas/iniciante/1190/1190.swift
import Foundation

let t = readLine(strippingNewline: true)!

var m = [[Double]](repeating: [Double](repeating: 0, count: 12), count: 12)
for i in 0..<12 {
  for j in 0..<12 {
    m[i][j] = Double(readLine(strippingNewline: true)!)!
  }
}

var media = 0.0
var quantidade = 0
for i in 0..<12 {
  for j in max(i+1,12-i)..<12 {
    media += m[i][j]
    quantidade += 1
  }
}

if (t == "M") {
  media /= Double(quantidade)
}

print(String(format: "%.1f", media))

File: problemas/iniciante/1190/1190.kt
import java.util.*

fun main(args: Array<String>) {
  val sc = Scanner(System.`in`)

  val t = sc.next()

  var M = Array(12) { FloatArray(12) }
  for (i in 0..11) {
    for (j in 0..11) {
      M[i][j] = sc.next().toFloat()
    }
  }

  var media = 0.0
  var quantidade = 0
  for (i in 0..11) {
    for (j in maxOf(i + 1, 12 - i)..11) {
      media += M[i][j]
      quantidade += 1
    }
  }

  if (t.equals("M")) {
    media /= quantidade
  }

  println(String.format("%.1f", media))
}
